### PR TITLE
switch to latest ruby for puppet6 + support latest bundler release

### PR DESCRIPTION
**Pull request by bastelfreak**
This got tested at https://github.com/voxpupuli/puppet-zabbix/pull/587

### CODE REVIEW

File: config_defaults.yml [hunk 1]
```diff
@@ -22,10 +22,10 @@
...
     bundler_args: --without system_tests development release
-  - rvm: 2.5.1
+  - rvm: 2.5.3
     env: PUPPET_VERSION="~> 6.0" CHECK=test_with_coveralls
     bundler_args: --without system_tests development release
-  - rvm: 2.5.1
+  - rvm: 2.5.3
     env: PUPPET_VERSION="~> 6.0" CHECK=rubocop
     bundler_args: --without system_tests development release
   - rvm: 2.4.4
```
Comment (dhoppe) [inline config_defaults.yml]: We should also update this version as long as the sync of the modules has not been done yet. The latest available stable version is 2.4.5.

File: config_defaults.yml [hunk 2]
```diff
@@ -3,17 +3,17 @@
   docker_sets:
   docker_defaults:
     # values will replace @@SET@@ with the docker_sets' value
-  - rvm: 2.5.1
+  - rvm: 2.5.3
```
Comment (ekohl) [inline config_defaults.yml]: If you always want the latest 2.5, why don't we just use `2.5`?

Comment (bastelfreak) [reply config_defaults.yml]: The original idea was to use the same ruby version as in the AIO version. But I'm not sure how useful that actually is.

Comment (ekohl) [reply config_defaults.yml]: In this case it doesn't matter because here we only launch a docker container

**Pull request merged**